docs/source/collect/logs.md

```diff
@@ -62,6 +62,11 @@ For duration string format see [time.ParseDuration](https://pkg.go.dev/time#Pars
 ##### `limits.maxLines`
 The number of lines to include, starting from the newest.
 
+##### `limits.maxBytes`
+The maximum file size of a collected pod log, is currently set to 5MB as this will ensure a support bundle will not grow excessively due
+to large logfiles. Neither will the logs contain information that is too old or no longer relevant.
+This value can be set in increments of bytes, e.g. a value of 5MB should be set as `5000000`.
```

Comment:
Just a suggestion, but I don't think all the background about the logs getting too large is needed in the description of what the field does. Seems fairly self-explanatory what you would use it for, and the descriptions of other fields tend to be briefer.
```suggestion
The maximum file size of a collected pod log. Defaults to `5000000`, which is 5MB.
```

Reply:
Agree, thanks @ajp-io